src/sentry/event_manager.py

```diff
@@ -486,6 +487,10 @@ def save(self, project, raw=False):
                 del tags['user']
             tags['sentry:user'] = event_user.tag_value
 
+        # At this point we want to normalize the in_app values in case the
```

Comment:
normalization now happens right before storing. This means that processors get the chance to only provide `in_app` if the client didn't do it yet.